server/src/test/java/org/opensearch/search/SearchServiceTests.java

```diff
@@ -225,9 +225,14 @@ public void onQueryPhase(SearchContext context, long tookInNanos) {
         }
     }
 
+    @Override
+    protected Settings featureFlagSettings() {
+        return Settings.builder().put(FeatureFlags.CONCURRENT_SEGMENT_SEARCH, true).build();
```

Comment:
this should also follow the same convention as other tests (missing the initialization with `super.featureFlagSettings()`)

` Settings.builder().put(super.featureFlagSettings()).put(FeatureFlags.CONCURRENT_SEGMENT_SEARCH, "true").build();`